store/migration/dev/20221126000000##principal_type.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE principal DROP CONSTRAINT principal_type_check;
+ALTER TABLE principal ADD CONSTRAINT principal_type_check CHECK (type IN ('END_USER', 'SYSTEM_BOT', 'SERVICE_ACCOUNT'));
```

Comment:
This is a false alert, I filed https://linear.app/bytebase/issue/BYT-1915/add-type-should-not-report-backward-compatibility-violation  @rebelice